src/librustc_error_codes/error_codes/E0271.md

```diff
@@ -14,11 +14,6 @@ impl Trait for i8 { type AssociatedType = &'static str; }
 foo(3_i8);
 ```
 
-This is because of a type mismatch between the associated type of some
-trait (e.g., `T::Bar`, where `T` implements `trait Quux { type Bar; }`)
-and another type `U` that is required to be equal to `T::Bar`, but is not.
-Examples follow.
```

Comment:
Why was this removed, i.e., how did it make you confused?

Reply:
It felt like this comment was (and it actually is) a left over of an older version. 
Reasons I don't like it:
* It tries to summarize the issue with completely different dummy words.
* It simply states the error message ("This is because of a type mismatch between the associated type of some
trait…)

So basically the first half of the text is contained in the error message itself, the second is explained (better and clearer imho) in the erroneous code example. No need to have this filler text there.

Reply:
Okay, it makes sense. Then we could also remove the first snippet? I think it's just a meaningless duplication now.

Reply:
It made me read the code twice and helped me understand the issue better. That's why I left it there. But you are right. Thanks for reviewing :+1: